### ISSUE TITLE
🔹 Step 3: Model Training

### ISSUE THREAD

**Issue by fayazmalik503** [fayazmalik503/Python_Projects_2025#3]
4 models use karna:

Logistic Regression ✅

Random Forest ✅

XGBoost ✅

LightGBM ✅

Accuracy + Confusion Matrix dekhna
